packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,130 @@
+### 1: The user navigates to the sign-in page
+
+When the user navigates to your app's sign-in page, call `OktaAuth.idx.start()` to start the sign-in flow.
+
+```javascript
+const { oktaAuth } = useOktaAuth();
+const startFlow = useCallback(async () => {
+  newTransaction = await oktaAuth.idx.start();
+}, [oktaAuth, flow, setTransaction]);
+```
+
+The method returns an `IdxTransaction` object containing field metadata that can be used to create a sign-in page dynamically.
+
+```json
+{
+   "status":"PENDING",
+   "nextStep":{
+      "name":"identify",
+      "inputs":[
+         {
+            "name": "username",
+            "label": "Username",
+            "required": true
+         }
+         ...
+      ]
+   },
+}
+```
+
+Display &mdash; either statically or dynamically using `IdxTransaction` &mdash; an input field for the user's username.
+
+<div class="half border">
+
+![Screenshot showing a sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+</div>
+
+### 2: The user submits their username
+
+When the user submits their account details, create an object with a `username` property and assign it the value entered by the user.
```

Comment:
Is it worth saying here that their username is also their email address?
```suggestion
When the user submits their username, create an object with a `username` property and assign it the value entered by the user.
```

Reply:
Updated to username. Confusingly, when using the Embedded SDK, on signup the username is the username and primary email address.  During sign in the username doesn't have to be the primary email address since the account could have been created not using the Embedded SDK (in the case of using the Admin console).